queries/dune_v2/period_slippage.sql

```diff
@@ -0,0 +1,497 @@
+            -- median.decimals,
+            intrinsic.decimals
+        ) as decimals,
+        COALESCE(
+            precise.price,
+            -- median_price,
+            intrinsic.price
+        ) as price
+    from token_times tt
+    LEFT JOIN precise_prices precise
+        ON precise.hour = tt.hour
+        AND precise.contract_address = token
+    -- LEFT JOIN prices.prices_from_dex_data median
+    --     ON median.hour = tt.hour
+    --     and median.contract_address = token
+    LEFT JOIN intrinsic_prices intrinsic
+        ON intrinsic.hour = tt.hour
+        and intrinsic.contract_address = token
+),
+-- ETH Prices: https://dune.com/queries/1578626?d=1
+eth_prices as (
+    select
+        date_trunc('hour', minute) as hour,
+        avg(price) as eth_price
+    from prices.usd
+    where blockchain = 'ethereum'
+    and contract_address = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+    and minute between '{{StartTime}}' and '{{EndTime}}'
+    group by date_trunc('hour', minute)
+),
+results_per_tx as (
+    select
+        ftbs.hour,
+        solver_address,
+        sum(token_imbalance_wei * price / pow(10, p.decimals)) as usd_value,
+        sum(token_imbalance_wei * price / pow(10, p.decimals) / eth_price) * pow(10, 18) as eth_slippage_wei,
+        tx_hash
+    from
+        final_token_balance_sheet ftbs
+    left join prices p
+        on token = p.contract_address
+        and p.hour = ftbs.hour
+    left join eth_prices ep
+        on  ftbs.hour = ep.hour
+    group by
+        ftbs.hour,
+        solver_address,
+        tx_hash
+    having
+        bool_and(price is not null)
+),
+-- Benchmark: 10 minutes for ONE DAY with non-pro account.
+-- select * from results_per_tx limit 10
+results as (
+    select
+        solver_address,
+        concat(environment, '-', name) as solver_name,
+        sum(usd_value) as usd_value,
+        sum(eth_slippage_wei) as eth_slippage_wei
+    from
+        results_per_tx rpt
+    join eth_prices ep
+        on rpt.hour = ep.hour
+    join cow_protocol_ethereum.solvers
+        on address = solver_address
+    group by
+        solver_address,
+        solver_name
+)
+select * from results
+
+-- -- select
+-- --     block_time,
+-- --     rpt.solver_name,
+-- --     concat('<a href="https://dune.com/queries/663231?TxHash=', concat('0x', encode(rpt.tx_hash, 'hex')), '" target="_blank">link</a>') as token_breakdown,
+-- --     concat('0x', encode(rpt.tx_hash, 'hex')) as tx_hash,
+-- --     usd_value,
+-- --     batch_value,
+-- --     100 * usd_value / batch_value as relative_slippage
+-- -- from results_per_tx rpt
+-- -- join gnosis_protocol_v2."batches" b
+-- --     on rpt.tx_hash = b.tx_hash
+-- -- where (
+-- --     abs(usd_value) > '{{MinAbsoluteSlippageTolerance}}'
+-- --     and
+-- --     100.0 * abs(usd_value) / batch_value > '{{RelativeSlippageTolerance}}'
+-- -- ) or
+-- --     abs(usd_value) > '{{SignificantSlippageValue}}'
+-- -- order by relative_slippage
```

Comment:
Forgotten query?

Reply:
This is actually something we may need to keep around for testing after... but it's been a while I can't exactly remember.